src/Compilers/Core/Portable/Diagnostic/LocalizableResourceString.FixedLocalizableString.cs

```diff
@@ -15,7 +15,7 @@ private sealed class FixedLocalizableString : LocalizableString
 
             private readonly string _fixedString;
 
-            public static FixedLocalizableString Create(string fixedResource)
+            public static new FixedLocalizableString Create(string fixedResource)
```

Comment:
>public static new FixedLocalizableString Create(string fixedResource) [](start = 12, length = 69)

Would it make sense to move this method as private to the base class instead?

Reply:
Yes, but wouldn't that be a breaking public API change?

Reply:
> Yes, but wouldn't that be a breaking public API change?

This class is private. How can making changes in it be a breaking public API change? 

---
In reply to: [292203172](https://github.com/dotnet/roslyn/pull/36067#discussion_r292203172) [](ancestors = 292203172)

Reply:
Ah, sorry my oversight. I looked at the existing code and moving this method to the base class would need us to change its private constructor and the private field `s_empty` to internal. I am instead going to just rename this method to `CreateFixed` and avoid the `new` modifier.